test/e2e/framework/pod.go

```diff
@@ -46,6 +51,26 @@ func WaitPodPresentOnClusterFitWith(cluster, namespace, name string, fit func(po
 	}, pollTimeout, pollInterval).Should(gomega.Equal(true))
 }
 
+func WaitPodMetricsReady(kubeClient kubernetes.Interface, karmadaClient karmada.Interface, cluster, namespace, name string) {
```

Comment:
a simple code comment of function would be better, just as other functions in this file

Reply:
done~